src/components/pages/Contacts/index.tsx

```diff
@@ -78,6 +83,13 @@ const UnconnectedContacts: React.FC<PropsFromRedux> = ({
   const [showTagModal, setShowTagModal] = useState<boolean>(false);
   const [selectedTags, setSelectedTags] = useState<Tag[]>([]);
 
+  const [showSendModal, setShowSendModal] = useState<boolean>(false);
```

Comment:
i dont think we need this extra boolean, we can just check for selectedContact

Reply:
yes sounds good - got rid of bool